connect_ext_ppr/models/utils.py

```diff
@@ -4,35 +4,35 @@
 
 def add_deployments(installation, listings, config, logger):
     with get_db_ctx_manager(config) as db:
-        print(db.bind)
         deployments = []
+        seen = set()
         for li in listings:
             product_id = li['product']['id']
             for hub in li['contract']['marketplace']['hubs']:
```

Comment:
I am not sure that the listing object provides hub details under the marketplace property.
<img width="820" alt="image" src="https://github.com/cloudblue/connect-extension-xvs/assets/1707894/5b48aeb2-21d5-4001-a668-3f93a7dd048d">

Reply:
Not but at this point I have all information needed processed by `get_all_info` function, I inject hub information there